### PR TITLE
Added documentation for the Near Cache preloader

### PR DESCRIPTION
**Pull request by Donnerbart**

### CODE REVIEW

File: src/NearCache.md [hunk 1]
```diff
@@ -107,6 +118,11 @@ Following are the descriptions of all configuration elements:
 - `local-update-policy`: Specifies the update policy of the local Near Cache. Is just available on JCache clients. Available values are as follows:
    - `INVALIDATE`: Does not update the local Near Cache. Will invalidate the local Near Cache eventually (default value).
    - `CACHE`: Updates the local Near Cache immediately after the put operation completes.
+- `preloader`: Specifies if the Near Cache should store and pre-load its keys for a faster re-population after a Hazelcast client restart. Is just available on IMap and JCache clients. It has the following attributes:
+  - `enabled`:
+  - `file-name`:
+  - `store-initial-delay-seconds`:
+  - `store-interval-seconds`:
```
Comment (Serdaro) [inline src/NearCache.md]: Specifies if the Near Cache should store and preload its keys for a faster re-population after a Hazelcast client restart. It is available only for IMap and JCache clients and has the following attributes:

??? the attributes need explanations, or claimed to be self-explanatory ???

File: src/NearCache.md [hunk 2]
```diff
@@ -298,3 +314,11 @@ You can configure eventual consistency with the system properties below (same pr
 - `hazelcast.invalidation.max.tolerated.miss.count`: Default value is 10. If missed invalidation count is bigger than this value, relevant cached data will be made unreachable. 
 - `hazelcast.invalidation.reconciliation.interval.seconds`: Default value is 60 seconds. This is a periodic task that scans cluster members periodically to compare generated invalidation events with the received ones from Near Cache.
 
+### Near Cache Preloader
+
+The Near Cache preloader is a functionality to store the keys from a Near Cache to provide a fast re-population of the previous hot data set after a Hazelcast Client has been restarted. It is available on IMap and JCache clients.
+
+The Near Cache preloader stores the keys (not the values) of Near Cache entries in regular intervals. You can define the initial delay via `store-initial-delay-seconds`, e.g. if you know that your hot data set will need some time to build up. You can configure the interval via `store-interval-seconds` which determines how often the key-set will be stored. The persistence will not run continuously. Whenever the storage is scheduled, it will be performed on the actual keys in the Near Cache.
+
```
Comment (Serdaro) [inline src/NearCache.md]: ..`store-initial-delay-seconds`, e.g., if you know.....

...how often the key set will be.....

File: src/NearCache.md (hunk 1 above)
Comment (Donnerbart) [inline src/NearCache.md]: Oh, will add the explanations.

File: src/NearCache.md (hunk 2 above)
Comment (Donnerbart) [inline src/NearCache.md]: We need a comma after each `e.g.`?

Comment (Serdaro) [inline src/NearCache.md]: Yes..."e.g." and "i.e." needs comma before and after.

**Comment by Donnerbart** [comment]
All comments have been addressed.

**Comment by Serdaro** [comment]
Thank you @Donnerbart

**Pull request merged**